api/server/server.go

```diff
@@ -430,6 +430,13 @@ func getContainersStats(eng *engine.Engine, version version.Version, w http.Resp
 	}
 	name := vars["name"]
 	job := eng.Job("container_stats", name)
+
+	stream := r.Form.Get("stream")
+	if stream == "" {
```

Comment:
@tiborvass It shouldn't be defaulting to false because of this right here.
Tested again just to make sure.